src/test/java/redis/clients/jedis/tests/commands/ControlCommandsTest.java

```diff
@@ -131,7 +131,8 @@ public void configGetSetBinary() {
     List<byte[]> info = jedis.configGet(maxmemory);
     assertArrayEquals(maxmemory, info.get(0));
     byte[] memory = info.get(1);
-    assertArrayEquals("OK".getBytes(), jedis.configSet(maxmemory, memory));
+    assertEquals("OK", jedis.configSet(maxmemory, memory));
+    assertEquals("OK", jedis.configSet(maxmemory, memory));
```

Comment:
```suggestion
    assertEquals("OK", jedis.configSetBinary(maxmemory, memory));
```
Why do you test the same method twice?

Reply:
@gkorland first one should be `200`. fixed.